packages/server/lib/modes/record.js

```diff
@@ -3,6 +3,7 @@ const la = require('lazy-ass')
 const chalk = require('chalk')
 const check = require('check-more-types')
 const debug = require('debug')('cypress:server:record')
+const logCommitInfo = require('debug')('cypress:server:info')
```

Comment:
does this really need to have a separate debug namespace?

if it does, can we at least nest it in `cypress:server:record`? `cypress:server:info` is pretty generic, i would prefer `cypress:server:record:ci-info` or something more to the point

Reply:
well, maybe, but to me the commit info is not really super tied to the record feature, and it is mostly an artifact of the code placement, no? But in the interests of moving the feature forward, I yield and will change this

Reply:
renamed